mongoose_app/management/commands/seed.py

```diff
@@ -247,6 +265,7 @@ def print(s):
                     sale_trans.save()
                     sale_trans_id += 1
 
+                    status = choice(list(TransactionType))
```

Comment:
This does mean that non-board members are seeded to make product mutations. Maybe it might be nicer to do the product mutations in separate pass where the user is fixed as the board user?